src/main/java/com/example/botjournal/entity/User.java

```diff
@@ -0,0 +1,47 @@
+package com.example.botjournal.entity;
+
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import org.glassfish.grizzly.http.util.TimeStamp;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+@Entity(name = "users")
+@Getter
+@Setter
+@ToString
+
+public class User {
+    @Id
+    @Column(name = "id", unique = true)
+    private Long id;
+
+    @Column(name = "firstname", nullable = false)
+    private String firstname;
+
+    @Column(name = "lastname")
+    private String lastname;
+
+    @Column(name = "username", unique = true)
+    private String username;
+
+    @Column(name = "registration_date", nullable = false)
+    private LocalDateTime registrationDate;
```

Comment:
Add extra line below.

Reply:
Also, probably `Instant` is better in this case because `LocalDateTime` doesn't contain zone component. Instants are always UTC+0.

See this answer for more details - https://stackoverflow.com/questions/75492508/should-i-use-instant-or-datetime-or-localdatetime-in-java-entities